_layouts/base.html

```diff
@@ -1,18 +1,38 @@
 <!DOCTYPE html>
 <html lang="{{ site.lang | default: "en-US" }}">
   <head>
+    <!-- html meta tags -->
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="msapplication-TileColor" content="#00aba9">
+    <meta name="theme-color" content="#ffffff">
     <meta name="author" content="SimBricks Team" />
     <meta charset="utf-8">
+    <meta name="title" content="SimBricks - Modular Full-System Simulation"/>
+    <meta name="description" content="SimBricks is an open-source modular simulation framework enabling full end-to-end evaluation of modern heterogeneous systems by combining battle-tested simulators for different system components like machines (e.g. QEMU, gem5, Simics), hardware components (e.g. Verilator, Tofino, FEMU SSD), and networks (e.g. ns-3, OMNeT++). SimBricks is fast, scalable, accurate and deterministically connects and synchronizes simulators while being able to run unmodified full system stacks, including applications, operating systems such as Linux, and hardware RTL." />
```

Comment:
Can we please use title and description from `_config.yml`? Especially if they are going to pasted in repeatedly. Otherwise this is just a recipe for inconsistencies. :-)

See here for an example:
https://gitlab.mpi-sws.org/osnet/os.mpi-sws.org/-/blob/main/_config.yml?ref_type=heads#L6
https://gitlab.mpi-sws.org/osnet/os.mpi-sws.org/-/blob/main/_includes/footer.html?ref_type=heads#L6

Reply:
Also get domain and url from the config please :-)

Reply:
@FreakyPenguin is the description text content wise fine for you or any change suggestions?

Reply:
Text seems fine :-)